content/blog/2019-09-25-making-function-calls-work.md

```diff
@@ -0,0 +1,299 @@
++++
+title = "Bril()"
+extra.author = "Gautam Mekkat, Eashan Garg, and Sameer Lal"
+extra.bio = """
+"""
++++
+
+## Bril()
+
+### Goal
+Our goal for this project was to make function calls work. Functions reduce code duplication and allow programmers to abstract away common functionality. This simplifies the development and improves the readability of complex programs. To make functions even more useful, we introduce function parameters, return types, optional type annotations for parameters, nested function definitions, and a simple module system. We also offer the option to pass command-line arguments to the main function.
```

Comment:
While I appreciate the notion, I think your audience will already be sold on the idea that functions are a good thing.